gems/pending/fs/ext4/superblock.rb

```diff
@@ -223,6 +223,14 @@ def inodeSize
       isDynamic? ? @sb['inode_size'] : INODE_SIZE
     end
 
+    def isEnabled64Bit?
+      gotBit?(@sb['incompat_flags'], ICF_64BIT)
```

Comment:
Good point. If we want to memorize, how about to memorize groupDescriptorSize directly?

attr_reader :sectorSize, :blockSize, :groupDescriptorSize

@groupDescriptorSize ||= isEnabled64Bit? ? @sb['group_desc_size'] : GDE_SIZE 

because so far this 64bit flag is only used here